Couldn't install on Windows by pip

I couldn't install it by pip on windows, and received the error messages below;

  error: command 'C:\\Users\\XXX\\AppData\\Local\\Programs\\Common\\Microsoft\\Visual C++ for Python\\9.0\\VC\\Bin\\cl.exe' failed with exit status 2

If somebody suggest me, it would be great helpful!
Hiro

Got wrong on windows too.

```
Installing collected packages: webrtcvad
  Running setup.py install for webrtcvad ... error
    Complete output from command c:\python27\python.exe -u -c "import setuptools, tokenize;__file__='c:\\users\\yihui\\appdata\\local\\temp\\pip-build-ndebsl\\webrtcvad\\setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record c:\users\yihui\appdata\local\temp\pip-me4tv2-record\install-record.txt --single-version-externally-managed --compile:
    running install
    running build
    running build_py
    creating build
    creating build\lib.win32-2.7
    copying webrtcvad.py -> build\lib.win32-2.7
    running build_ext
    building '_webrtcvad' extension
    creating build\temp.win32-2.7
    creating build\temp.win32-2.7\Release
    creating build\temp.win32-2.7\Release\cbits
    creating build\temp.win32-2.7\Release\cbits\webrtc
    creating build\temp.win32-2.7\Release\cbits\webrtc\common_audio
    creating build\temp.win32-2.7\Release\cbits\webrtc\common_audio\signal_processing
    creating build\temp.win32-2.7\Release\cbits\webrtc\common_audio\vad
    C:\Users\Yihui\AppData\Local\Programs\Common\Microsoft\Visual C++ for Python\9.0\VC\Bin\cl.exe /c /nologo /Ox /MD /W3 /GS- /DNDEBUG -DWEBRTC_POSIX -Icbits -Ic:\python27\include -Ic:\python27\PC /Tccbits\pywebrtcvad.c /Fobuild\temp.win32-2.7\Release\cbits\pywebrtcvad.obj
    pywebrtcvad.c
    cbits\pywebrtcvad.c(24) : error C2275: 'PyObject' : illegal use of this type as
an expression
            c:\python27\include\object.h(108) : see declaration of 'PyObject'
    cbits\pywebrtcvad.c(24) : error C2065: 'vadptr' : undeclared identifier
    cbits\pywebrtcvad.c(25) : error C2065: 'vadptr' : undeclared identifier
    cbits\pywebrtcvad.c(97) : error C2143: syntax error : missing ';' before 'type'
    cbits\pywebrtcvad.c(104) : error C2065: 'result' : undeclared identifier
    cbits\pywebrtcvad.c(117) : error C2061: syntax error : identifier 'WebRtcVadMethods'
    cbits\pywebrtcvad.c(117) : error C2059: syntax error : ';'
    cbits\pywebrtcvad.c(117) : error C3409: empty attribute block is not allowed
    cbits\pywebrtcvad.c(117) : error C2513: '/*global*/ ' : no variable declared before '='
    cbits\pywebrtcvad.c(159) : error C2065: 'WebRtcVadMethods' : undeclared identifier
    cbits\pywebrtcvad.c(159) : warning C4047: 'function' : 'PyMethodDef *' differs in levels of indirection from 'int'
    cbits\pywebrtcvad.c(159) : warning C4024: 'Py_InitModule4' : different types for formal and actual parameter 2
    error: command 'C:\\Users\\Yihui\\AppData\\Local\\Programs\\Common\\Microsoft\\Visual C++ for Python\\9.0\\VC\\Bin\\cl.exe' failed with exit status 2
```

--- a/cbits/pywebrtcvad.c
+++ b/cbits/pywebrtcvad.c
@@ -18,10 +18,11 @@ static void vad_free(PyObject* vadptr)
 static PyObject* vad_create(PyObject *self, PyObject *args)
 {
   VadInst *handle;
+  PyObject *vadptr;
   if (WebRtcVad_Create(&handle)) {
     return NULL;
   }
-  PyObject *vadptr = PyCapsule_New(handle, "WebRtcVadPtr", vad_free);
+  vadptr = PyCapsule_New(handle, "WebRtcVadPtr", vad_free);
   return Py_BuildValue("O", vadptr);
 }
 
@@ -87,6 +88,7 @@ static PyObject* vad_process(PyObject *self, PyObject *args)
   long fs;
   Py_buffer audio_frame = {NULL, NULL};
   long frame_length;
+  int result;
 #ifdef PY3
   if (!PyArg_ParseTuple(args, "Oly*l", &vadptr, &fs, &audio_frame, &frame_length)) {
 #else
@@ -94,7 +96,7 @@ static PyObject* vad_process(PyObject *self, PyObject *args)
 #endif
     return NULL;
   }
-  int result =  WebRtcVad_Process(PyCapsule_GetPointer(vadptr, "WebRtcVadPtr"),
+  result =  WebRtcVad_Process(PyCapsule_GetPointer(vadptr, "WebRtcVadPtr"),
                                   fs,
                                   audio_frame.buf,
                                   frame_length);

--- a/setup.py
+++ b/setup.py
@@ -1,6 +1,7 @@
 from setuptools import setup, find_packages, Extension
 import glob
 import os.path
+import sys
 
 
 C_SRC_PREFIX = os.path.join('cbits', 'webrtc', 'common_audio')
@@ -19,8 +20,15 @@ c_sources = (
             'vad',
             '*.c')))
 
+define_macros = []
+
+if sys.platform.startswith('win'):
+    define_macros.extend([('_WIN32', None),])
+else:
+    define_macros.extend([('WEBRTC_POSIX', None),])
+
 module = Extension('_webrtcvad',
-                   define_macros=[('WEBRTC_POSIX', None)],
+                   define_macros=define_macros,
                    sources=c_sources,
                    include_dirs=['cbits'])
 

